state.go

```diff
@@ -23,6 +23,11 @@ import (
 const genesisEpoch uint64 = 1
 const epochInterval int64 = 5
 
+var (
+	errDocDescriptor = errors.New("insufficient descriptors uploaded")
```

Comment:
Maybe rename to `errDocInsufficientDescriptor`.

Reply:
Fixed